dev_docs/tutorials/debugging_fips_test_failures.mdx

```diff
@@ -0,0 +1,63 @@
+---
+id: kibDevTutorialDebuggingFipsTestFailures
+slug: /kibana-dev-docs/tutorials/debugging-fips-test-failures
+title: Debugging FIPS test failures
+description: Learn how to debug test failures generated from the FIPS nightly pipeline
+date: 2024-12-04
+tags: ['kibana', 'FIPS', 'dev', 'test', 'debugging', 'FTR']
+---
+
+## What is FIPS?
+
+The Federal Information Processing Standard (FIPS) is a set of standards that describe document processing,
+encryption algorithms, and other information technology standards for use within non-military government agencies 
+and by government contractors and vendors who work with the agencies.
+
+### How does that relate to Kibana?
+
+Kibana can be configured to run in a FIPS-compliant mode, which disallows certain configurations,
+requires a Platinum or better license, and requires the Security plugin to be enabled.
+
+In order to ensure parity between the FIPS-compliant mode and the standard mode,
+Kibana has a nightly pipeline that runs as many tests as possible to ensure that Kibana is FIPS compliant.
+
+### Why do we skip some tests?
+
+As Kibana's FIPS mode requires a Platinum or better license and the Security plugin to be enabled, Kibana's test suite configuration is overridden to meet these requirements.
+
+With these overrides, the assumptions that the tests make about the environment are no longer valid, and some tests fail. For instance, if a test is asserting that a certain feature is not
+available with a basic license, that test will fail in the FIPS pipeline as the license is forced to true.
+
+### How do we discern between test failures resulting from FIPS overrides and actually FIPS/OpenSSL errors?
+
+For an override related error the `expect(...)` will usually fail with typical errors, such as "expected 404, got 200", or it may be more subtle, but related to the subject of the test.
+
+See the image below for an example of a test failure due to the license type being overridden:
+![image][./fips_test_failure_license_override_example.png]
```

Comment:
Question: @jeramysoucy do you know if this will display properly in the docs? In the rich diff view it just shows the markdown

Reply:
I _think_ this will work. But I am not an MDX expert.